packages/application-components/src/components/dialogs/internals/dialog-container.tsx

```diff
@@ -109,6 +110,7 @@ const DialogContainer = (props: Props) => (
             css={css`
               min-height: 0;
               ${sizeStyles(props)}
+              padding: ${designTokens.paddingForDialogContainer};
```

Comment:
Dialogs are using the `Card` component which internally uses the `Spacing.Inquish` component. The latter does not have a scale which fits with the expected paddings for Dialogs so we use a custom padding here to add what it's missing from the Card paddings to meet the desired values.